configs/other_software/vcs/svn.yaml

```diff
@@ -1,2 +1,7 @@
-nothing:
-        - so far
+get_command: "svn co ${define_options} ${define_revision} ${repository}"
```

Comment:
Hooray for a finally useful `svn.yaml`! Most things are in git, but nevertheless, good that we can add it to the list of things we support

Reply:
which we don't need anymore, as the NEMO people finally switched to git a few weeks ago, hooray